src/reporter.ts

```diff
@@ -262,16 +328,25 @@ class RPReporter implements Reporter {
       status,
     } = this.findTestItem(this.testItems, test.title);
     let withoutIssue;
+    let descriptionWithError;
     if (result.status === STATUSES.SKIPPED) {
       withoutIssue = this.config.skippedIssue === false;
     }
 
+    if (result.error) {
+      this.sendLogOnFail(testItemId, result.error);
+      descriptionWithError = (description || '').concat(
+        `\n\`\`\`error\n${result.error.message}\n\`\`\``,
```

Comment:
Should we provide just message or full log into description? Logs in description and in the test are not the same